pkg/pod/pod.go

```diff
@@ -84,6 +85,7 @@ func RemoveFinalizersFromPod(c client.Client, logger logr.Logger, pl *corev1.Pod
 
 			err := c.Update(context.TODO(), &p)
 			if err != nil {
+				logger.Error(err, "failed to remove finalizer from the pod %v/%v", p.Namespace, p.Name)
```

Comment:
Same as earlier comment.

```suggestion
				logger.Error(err, fmt.Sprintf("failed to remove finalizer from the pod %v/%v", p.Namespace, p.Name))
```

or

```suggestion
				logger.Error(err, fmt.Sprintf("failed to remove finalizer from the pod", "namespace", p.Namespace, "name", p.Name))
```